src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1037,9 +1037,33 @@ public float CalculateHitpoints()
         return maxHitpoints;
     }
 
-    public float CalculateStorage()
+    public float GetNominalCapacity()
     {
-        return MicrobeInternalCalculations.CalculateCapacity(editedMicrobeOrganelles);
+        return editedMicrobeOrganelles.Sum(o => MicrobeInternalCalculations
```

Comment:
I think the internal calculations should still have a method that takes in an enumerable to calculate the nominal capacity, instead of needing to use LINQ here in the editor code.